workflow/circles.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import division
+
+import numpy as np
```

Comment:
maybe call this `_circles.py` to make it clear this is only used in one of the other files and does not actually contain a score definition ? (and the same for draw.py)